theme/theme.ts

```diff
@@ -0,0 +1,207 @@
+import { createTheme } from '@mui/material'
+
+const greenPalette = {
+  0: '#000000',
+  10: '#00201B',
+  17: '#003029',
+  20: '#00382F',
+  30: '#005045',
+  40: '#006B5D',
+  50: '#008675',
+  60: '#00A38E',
+  70: '#32BFA9',
+  80: '#57DBC4',
+  90: '#77F8DF',
+  95: '#B6FFEE',
+  99: '#F3FFFA',
+  100: '#FFFFFF',
+}
+
+const yellowPalette = {
+  0: '#000000',
+  10: '#1E1C00',
+  20: '#343200',
+  30: '#4B4900',
+  40: '#646100',
+  50: '#7F7A00',
+  60: '#9A9400',
+  70: '#B6AF02',
+  80: '#D2CB2D',
+  90: '#EFE84A',
+  92: '#F5ED4F',
+  95: '#FDF657',
+  98: '#F5ED4F',
+  99: '#F3FFFA',
+  100: '#FFFFFF',
+}
+
+const neutralPalette = {
+  25: '#343D3E',
+  98: '#F7FAFA',
+  197: '#F2F7F7',
+}
+
+export const designColor = {
+  white: '#F7FAFA',
+  black: '#00211F',
+  background: neutralPalette[197],
+  green: {
+    main: greenPalette[17],
+    light: greenPalette[30],
+    dark: greenPalette[20],
+  },
+  yellow: {
+    main: yellowPalette[92],
+    light: yellowPalette[98],
+  },
+}
+
+export const theme = createTheme({
+  palette: {
+    mode: 'light',
+    primary: {
+      main: designColor.green.main,
+      light: designColor.green.light,
+      dark: designColor.green.dark,
+      contrastText: designColor.white,
+    },
+    secondary: {
+      main: designColor.yellow.main,
+      contrastText: designColor.black,
+      light: designColor.yellow.light,
+    },
+    background: {
+      default: '#F2F7F7',
+    },
+  },
+  typography: {
+    fontFamily: 'Montserrat, Arial, sans-serif',
+    button: {
+      fontFamily: 'Montserrat',
+      fontStyle: 'bold',
+      fontWeight: 700,
+      fontSize: '14px',
+      lineHeight: '20px',
+      letterSpacing: '+0.1px',
+    },
+    displayLarge: {
```

Comment:
This is giving build error because many properties here are not in the type. Since it's TypeScript it's enforcing it. You'd need to use module argumentation per [MUI doc](https://mui.com/material-ui/customization/theming/#custom-variables)


```
interface FontProperties {
    fontFamily?: string
    fontWeight?: number,
    fontSize?: string,
    lineHeight?: string,
    letterSpacing?: string,
    fontStyle?: string,
}
declare module '@mui/material/styles/createTypography' {
  // allow configuration using `createTheme`
  interface TypographyOptions {
    displayLarge?: FontProperties,
    displayMedium?: FontProperties,
    displaySmall?: FontProperties,
    headlineLarge?: FontProperties,
    headlineMedium?: FontProperties,
    headlineSmall?: FontProperties,
    titleLarge?: FontProperties,
    titleMedium?: FontProperties,
    titleSmall?: FontProperties,
    labelLarge?: FontProperties,
    labelMedium?: FontProperties,
    labelSmall?: FontProperties,
    bodyLarge?: FontProperties,
    bodyMedium?: FontProperties,
    bodySmall?: FontProperties
  }
}

```

Reply:
Thanks for catching this and explaining the issue, @santina! I'm still getting used to Typescript. The ability to use interfaces to describe the shape of an object is exciting.